core/src/main/resources/hudson/logging/Messages_pt_BR.properties

```diff
@@ -24,3 +24,4 @@
 LogRecorderManager.init=Inicializando grava\u00e7\u00e3o de log
 # log
 LogRecorderManager.DisplayName=log
+Add a logger by this name to a log recorder before attempting to configure its level.
```

Comment:
This addition here looks weird

Reply:
Indeed, thanks @res0nance !
Is there any know automatic way to validate property files? That would avoid this one.